src/Ocelot/Configuration/Validator/RouteFluentValidator.cs

```diff
@@ -1,19 +1,21 @@
-﻿using System.Linq;
-using System.Text.RegularExpressions;
-using System.Threading;
-using System.Threading.Tasks;
-
-using Ocelot.Configuration.File;
-
-using FluentValidation;
-
+﻿using FluentValidation;
 using Microsoft.AspNetCore.Authentication;
+using Ocelot.Configuration.File;
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Ocelot.Configuration.Validator
 {
     public class RouteFluentValidator : AbstractValidator<FileRoute>
     {
         private readonly IAuthenticationSchemeProvider _authenticationSchemeProvider;
+        private static readonly Regex _secondsRegEx = new Regex("^[0-9]+s", RegexOptions.Compiled, TimeSpan.FromMilliseconds(100));
+        private static readonly Regex _minutesRegEx = new Regex("^[0-9]+m", RegexOptions.Compiled, TimeSpan.FromMilliseconds(100));
+        private static readonly Regex _hoursRegEx = new Regex("^[0-9]+h", RegexOptions.Compiled, TimeSpan.FromMilliseconds(100));
+        private static readonly Regex _daysRegEx = new Regex("^[0-9]+d", RegexOptions.Compiled, TimeSpan.FromMilliseconds(100));

```

Comment:
Hardcode has detected!
The `100` milliseconds constant should be public static.

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1348/commits/68ef3a51989074f8ca6da43269d6ec948ceb6544 ✔️